package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@alertlogic/al-collector-js",
-  "version": "2.0.4",
+  "version": "2.0.5",
```

Comment:
We don't need to bump package version because we only change travis config.

Reply:
Done, reverted to previous version